docs/proposals/mcs/remove-derived-mcs.md

```diff
@@ -0,0 +1,352 @@
+root@btg:/# cat /etc/resolv.conf
+search default.svc.cluster.local svc.cluster.local cluster.local clusterset.local
+nameserver 10.13.0.10
+options ndots: 5
+```
+
+This configuration method needs to configure the `/etc/resolv.conf` file on the client node. If you don't want to change the node's `resolv.conf` file, you can also use the following method.
+
+**Configure pod's dnsConfig in yaml**
+
+That is, add the configuration item searches of dnsConfig in yaml, and set the value to `clusterset.local`, as shown below:
+
+```yaml
+...
+       dnsPolicy: ClusterFirst
+       dnsConfig:
+         searches:
+           -clusterset.local
+         options:
+           - name: timeout
+             value: "2"
+```
+
+###### Access method
+
+Since the search function of dns is used, the `foo` service  access method is:
+
+1. `foo.default.svc` access. This method gives priority to local access, and if the local service with the same name is unavailable, it will access the remote cluster service.
+
+2. `foo.default.svc.clusterset.local` full domain name access. The request will go directly to the remote server for processing.
+
+###### Advantages and disadvantages
+
+shortcoming:
+
+1. Using `foo.default.svc` access cannot achieve load balancing. The remote service will only be accessed when the local service is unavailable. Therefore, if you need to achieve local and remote access load balancing, you need to cooperate with other solutions.
+
+2. To use `foo.default.svc`, you need to modify the dns configuration. If you use the `resolv.conf `method, you need to modify the `resolv.conf` file of each possible nodes. If you use the dnsConfig method, you need to modify the yaml or api of the delivered pod.
+
+3. The `multicluster` plug-in cannot be loaded dynamically. Customers need to recompile `coredns` and modify the `coredns` configuration of member clusters.
+
+4. The original access method will be changed. This solution uses different suffixes to distinguish local and remote services. Therefore, the original access method `foo.default.svc.cluster.local` will become `foo.default.svc.clusterset.local`. Customers need to modify the code to adapt.
+
+advantage:
+
+1. The impact is controllable. No new components will be added. The way coredns adds multicluster is to comply with the official proposal of coredns, and the code is open.
+
+2. The amount of modification is small. It can be done by using the existing code and scheme of multicluster.
+
+#### 2) Additional dns server
+
+Similar to submariner's [SERVICE DISCOVERY](https://submariner.io/getting-started/architecture/service-discovery/). It is necessary to develop an extended dns resolver, and install the corresponding plug-in in coreDNS to transfer the domain name with the specified suffix (such as `clusterset.local`) to this additional dns resolver. Therefore, the domain name can be resolved by dns.
+
+The workflow of the additional dnsvserver as follows:
+
+![](.\additional-dns-server.jpg)
+
```

Comment:
This is no image showing here:

![image](https://github.com/karmada-io/karmada/assets/78244870/08c30f26-5eb8-4d58-b816-0d40c82cbdcc)

Reply:
> This is no image showing here:
> 
> ![image](https://user-images.githubusercontent.com/78244870/253713218-08c30f26-5eb8-4d58-b816-0d40c82cbdcc.png)

oh， it's ok in my brower. I think you need a pull, for this is l just up recently.